internal/runner/detector.go

```diff
@@ -55,6 +55,6 @@ func DetectRunner(cfg config.Config) (TestRunner, error) {
 		return NewCucumber(runnerConfig), nil
 	default:
 		// Update the error message to include the new runner
-		return nil, errors.New("runner value is invalid, possible values are 'rspec', 'jest', 'cypress', 'playwright', 'pytest', 'pytest-pants', 'gotest', or 'cucumber'")
+		return nil, fmt.Errorf("runner value %q is invalid, possible values are 'rspec', 'jest', 'cypress', 'playwright', 'pytest', 'pytest-pants', 'gotest', or 'cucumber'", testRunner)
```

Comment:
Previously the caller was prepending the name of the invalid runner to the error message https://github.com/buildkite/test-engine-client/blob/45db707f44cb53ef92da660edcc41bf3fb155f0c/main.go#L113

With this change the caller error wrapping is simplified.
```go
return fmt.Errorf("Unsupported value for BUILDKITE_TEST_ENGINE_TEST_RUNNER: %v", err)
```